backend/src/main/scala/cromwell/backend/ReadLikeFunctions.scala

```diff
@@ -8,7 +8,7 @@ import scala.concurrent.Future
 import scala.util.Try
 
 trait ReadLikeFunctions extends PathFactory with IoFunctionSet with AsyncIoFunctions {
-  
+
```

Comment:
Consider reverting. IMHO no real reason to include this file in this change.

Reply:
turns out "just whitespace" gets ignored by my `git` version. I tried as part of `More whitespace reverting`...